bundles/org.openhab.binding.shelly/src/main/resources/ESH-INF/thing/device.xml

```diff
@@ -8,10 +8,63 @@
 		<label>Device Status</label>
 		<description>Some details on the device status</description>
 		<channels>
-			<channel id="uptime" typeId="uptime" />
-			<channel id="wifiSignal" typeId="system.signal-strength" />
 			<channel id="alarm" typeId="alarmTrigger" />
+			<channel id="wifiSignal" typeId="system.signal-strength" />
+			<channel id="uptime" typeId="uptime" />
 		</channels>
 	</channel-group-type>
 
+	<channel-type id="alarmTrigger">
+		<kind>trigger</kind>
+		<label>Alarm</label>
+		<description>Alarm Trigger, e.g. weak WiFi Signal detected or over heating.</description>
+		<event>
+			<options>
+				<option value="NONE">None</option>
+				<option value="WEAK_SIGNAL">Weak WiFi</option>
+				<option value="RESTARTED">Device restarted</option>
+				<option value="OVERTEMP">Device is overheating</option>
+				<option value="OVERLOAD">Device is overloaded</option>
+				<option value="OVERPOWER">Device is over max power</option>
+				<option value="LOAD_ERROR">Load error</option>
+				<option value="LOW_BATTERY">Low battery</option>
+				<option value="BATTERY">Wakeup by battery</option>
+				<option value="POWERON">Device was powered on</option>
+				<option value="BUTTON">Button was pressed</option>
+				<option value="SENSOR">Sensor data updated</option>
+			</options>
+		</event>
+	</channel-type>
+	<channel-type id="charger">
+		<item-type>Switch</item-type>
+		<label>Charger connected</label>
+		<description>ON: Device is charging, OFF: No charger connected</description>
+		<state readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="batVoltage" advanced="true">
+		<item-type>Number:ElectricPotential</item-type>
+		<label>Battery voltage</label>
```

Comment:
The first letter of every word in a label should be capitalized.
```suggestion
		<label>Battery Voltage</label>
```

Reply:
changed